tests/baselines/reference/docker/test262.log

```diff
@@ -0,0 +1,11028 @@
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.1.1_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.1.1_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A3.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A4.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.2.1_A4.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A2.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A2.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S15.6.3_A3.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A1_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A1_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A2_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A2_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A3_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A3_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T1.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T1.js (strict mode)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T3.js (default)
+  Expected no error, got SyntaxError: Unexpected token ,
+FAIL test/built-ins/Boolean/S9.2_A4_T3.js
```

Comment:
I'm _pretty sure_ these CJK failures are because while we handle CJK just fine, we don't downlevel it in any way (we just pass it thru), and node `0.10` does _not_ handle CJK just fine. Meaning these failures are maybe expected?